python/sauron-core/core/analysis/stats.py

```
@@ -0,0 +1,197 @@
+import numpy as np
+import pandas as pd
+from typing import Dict, List, Tuple
```

Ruff (0.8.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)